code/game/objects/structures/bearcave.dm

```diff
@@ -0,0 +1,188 @@
+/obj/structure/bearcave
+	name = "Bear Cave"
+	icon = 'icons/obj/bearcave.dmi'
+	icon_state = "cave_den"
+	desc = "Thats a bearcave. You probably want to stay away from it. "
+	var/struc_health = 1000
```

Comment:
This is a good PR in general, but what will this value eventually be?

It might be worth considering other similar proc structures like tree's health & what's currently programmed into grandfather clocks /(or structures that have proper damage tracking), and ot make the bear cave inflammable but in its present state this pr is fine to be shipped off.

Reply:
That was me trying to PR before going to bed. There are no plans for that health var.
I'll clean the unused vars and commit it up. 
Thanks for pointing it out